packages/messages-react/src/SystemMessage.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactNode } from "react";
+import classNames from "classnames";
+import { MessageIcon } from "./common/MessageIcon";
+import { messageTypes } from "./common/types.d";
+
+interface Props {
+    messageType: messageTypes;
+    children: ReactNode;
+    className?: string;
+    inverted?: boolean;
+    maxWidth?: string;
+    paddingLeft?: string;
+}
+
+export const SystemMessage: React.FC<Props> = ({
+    messageType,
+    children,
+    className = "",
+    inverted,
+    maxWidth,
+    paddingLeft,
+}) => {
+    const componentClassName = classNames("jkl-system-message", "jkl-system-message--" + messageType, className, {
+        "jkl-system-message--dark": inverted,
+    });
+
+    const styles = Object.entries({
```

Comment:
Add a comment about what's going on here and why

Reply:
Likte vel egentlig ikke dette. Dette burde bare løses ved å sende inn en klasse. Hvis vi ønsker at det skal være noen defaulter, så kan vi godt legge til det som props, men ikke enig i at dette fortjener å være så tilgjengelig og inline style.

Reply:
Med klasser blir det vanskelig å støtte oppunder de forskjellige løsningene folk har valgt for plassering av innhold.
Om jeg ikke tar helt feil sier ikke designsystemet noe om hva slags bredder som skal brukes på innhold, og det er da vanskelig å vite hvordan innhold skal offsettes.

Reply:
Ser ikke helt hvorfor det blir vanskeligere å sende inn klasse enn tallene. Jeg tenker det oppfordrer til å gjennbruke klassene man har lagt og forhåpentligvis sørger for at hver enkelt løsning er konsekvent. Det sagt, så er jeg litt usikker på hvorfor det skal være så ulike behov for padding/bredder, burde være mulig å standardisere, og evt tillate overrideing med klassen.

Reply:
Har du noen innspill her @Mikkelinski? Spesielt med tanke på standardisering av innholdsbredder, plassering på sider, hva slags spacing som må til for å aligne en system melding med resten av innholdet osv.
Er det mulig å standardisere dette med klasser på tvers av systemer, eller vil det v're for mye spesialbehov?

Reply:
Tror fort dette blir problematisk å standarisere - vi har spacing skalaen vår for å blokke ut layouts, men vi er ganske forsiktige i å snakke om bredde / kolonneoppsett fordi vi har såpass mange ulike løsninger som kjører det veldig forskjellig (ikke alle responsive engang).

Jeg tror fort at man bør ha eksempler som illustrerer bruken av komponenten(e) og beskrivelser av bruk fremfor å fremtvinge noen standarder.

Reply:
Ja, tror selv dette kan bli vanskelig.

Kunne kanskje du og @Murstam hjulpet til med å lage noen eksempler? Do's and dont's for `AlertMessage`.

Reply:
Jepp, det kan vi fikse!